tests/functional/ui/test_health_overview.py

```
@@ -600,3 +701,128 @@ def test_health_score_with_blackbox_and_metrics_exporter_shutdown(
+        """
+        page_nav = PageNavigator()
+        df_overview = page_nav.nav_storage_data_foundation_overview_page()
+        baseline_score = self.get_health_score_ui()
+        logger.info(f"Baseline health score from overview: {baseline_score}")
+        page_nav.take_screenshot("initial_health_overview")
+        infra_health_overview = df_overview.nav_health_view_checks()
+        logger.info("Verifying alerts in Active Alerts section")
+        infra_health_overview.click_last_24_hours_alerts()
+        all_alerts = infra_health_overview.get_all_checks()
+        unique_alert_types = list(
+            set(alert.check for alert in all_alerts if alert.end_time is None)
```

Ruff (0.15.15)

[warning] Unnecessary generator (rewrite as a set comprehension)

Rewrite as a set comprehension

(C401)